README.md

```diff
@@ -117,36 +119,47 @@ badges:
-the `main` branch. If in doubt, make sure to talk about your contribution to the team, either via an
-issue or [Discord](https://discord.gg/pxrBmy4).
 
-Any help is appreciated! Comments, suggestions, issues, PRs.
+## Contributing
 
 Have you found a bug or have a suggestion of how to enhance Flame? Open an issue and we will take a
 look at it as soon as possible.
 
-Do you want to contribute with a PR? PRs are always welcome, just be sure to create it from the
-orrect branch (see above) and follow the [checklist](.github/pull_request_template.md) which will
+Do you want to contribute with a PR? PRs are always welcome, just make sure to create it from the
+correct branch (main) and follow the [checklist](.github/pull_request_template.md) which will
 appear when you open the PR.
 
+If in doubt, make sure to talk about your contribution to the team, either via an issue or
+[Discord](https://discord.gg/pxrBmy4).
+
+
 ## Getting started
 
-A collection of guides can be found [here](./tutorials). Note that these tutorials are based on the
-main branch. To make sure you are looking at the tutorials that work with your current version,
-select the correct version tag.
+A simple tutorial to get started can be found on
+[tutorials.flame-engine.org](https://tutorials.flame-engine.org) and examples of most features in
+Flame can be found on [examples.flame-engine.org](https://examples.flame-engine.org). To access the
+code for each example, press `< >` in the top right corner.
+
+
+### External tutorials
 
-This collection of guides is a work in progress, more guides and tutorials will be added soon.
+- @Devowl's Flutter & Flame series: 
+  - Step 1: Create your game - https://medium.com/flutter-community/flutter-flame-step-1-create-your-game-b3b6ee387d77
+  - Step 2: Game basics - https://medium.com/flutter-community/flutter-flame-step-2-game-basics-48b4493424f3
+  - Step 3: Sprites and inputs - https://blog.devowl.de/flutter-flame-step-3-sprites-and-inputs-7ca9cc7c8b91
+  - Step 4: Collisions & Viewport - https://blog.devowl.de/flutter-flame-step-4-collisions-viewport-ff2da048e3a6
+  - Step 5: Level generation & camera - https://blog.devowl.de/flutter-flame-step-5-level-generation-camera-62a060a286e3 
 
-We also offer a curated list of Games, Libraries and Articles over at
+- Other tutorials:
+  - Building Games in Flutter with Flame - [Getting Started](https://www.raywenderlich.com/27407121-building-games-in-flutter-with-flame-getting-started)
+  - @DevKage's YouTube series with the [Dino run tutorial](https://www.youtube.com/playlist?list=PLiZZKL9HLmWOmQgYxWHuOHOWsUUlhCCOY)
+
+We offer a curated list of Games, Libraries and Articles over at
 [awesome-flame](https://github.com/flame-engine/awesome-flame).
 
 Note that some of the articles might be slightly outdated, but they are still useful.
 
+
 ## Credits
 
  - The [Flame Engine team](https://github.com/orgs/flame-engine/people), who are continuously
```

Comment:
blue fire?
+ our pratrons?
+ our sponsors?
maybe join with sponsor section? or just remove?

Reply:
I think it is better to have the sponsors sections separate so that they can see that they stand-out if they sponsor us.

Updating to the Blue fire team instead.